beacon-chain/monitor/service.go

```diff
@@ -0,0 +1,34 @@
+package monitor
+
+import (
+	types "github.com/prysmaticlabs/eth2-types"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	log = logrus.WithField("prefix", "monitor")
+)
+
+// ValidatorMonitorConfig contains the list of validator indices that the
+// monitor service tracks, as well as the event feed notifier that the
+// monitor needs to subscribe.
+type ValidatorMonitorConfig struct {
+	TrackedValidators []types.ValidatorIndex
```

Comment:
Consider using a map for O(1) lookups. 

```suggestion
	TrackedValidators map[types.ValidatorIndex]interface{}
```

You can still iterate over it like a slice, but `TrackedIndex` becomes much simpler.
```go
func (s *Service) TrackedIndex(idx types.ValidatorIndex) bool {
  _, ok := s.trackedValidators[idx]
  return ok
}
```